x-pack/test/security_solution_api_integration/test_suites/detections_response/utils/combine_to_ndjson.ts

```diff
@@ -8,3 +8,7 @@
 export function combineToNdJson(...parts: unknown[]): string {
   return parts.map((p) => JSON.stringify(p)).join('\n');
 }
+
+export function combineArraysToNdJson(...arrays: unknown[][]): string {
```

Comment:
True, that would work also. I try to avoid using the spread operator out of habit for arrays of more than a few items since it copies the items inside. I only went halfway initially though by using `combineToNdJson` internally so the arrays were still being spread. I added a new function to handle array inputs specifically without spreading.